src/components/App.tsx

```diff
@@ -2421,21 +2462,12 @@ class App extends React.Component<any, AppState> {
           this.state,
         );
         if (selectedElements.length > 0) {
-          const { x, y } = viewportCoordsToSceneCoords(
-            event,
-            this.state,
-            this.canvas,
-            window.devicePixelRatio,
+          const [pointerX, pointerY] = pointOnGrid(
```

Comment:
`pointerX` is no longer a good name (it wasn't good from the beginning.)
Suggestions are welcome.

Reply:
in my latest PRs I'm migrating to `viewportX` and `sceneX` so that we know what coord system we're in. Probably not what you need here though.

Reply:
This is the new point of the top left the dragging element(s) bounds. 🤔

Reply:
I'd just call it `newX` or `gridX` --- though it won't be quite obvious that if grid is off, it won't change the values, but it's not really obvious even now.

You can also cop out of naming altogether (but this functional approach may be unreadable):

```
if (
  resizeElements(
    resizeHandle,
    setResizeHandle,
    selectedElements,
    resizeArrowDirection,
    event,
    ...pointOnGrids(
      x - resizeOffsetXY[0],
      y - resizeOffsetXY[1],
      this.state.gridSize,
    )
  )
) {
  return;
}
```

Reply:
Btw the `pointOnGrids` is not very descriptive. Maybe `getGridPoint`, or `toGridPoint`, or `getPointOnGrid`?

Reply:
Thanks for your suggestion!